### ISSUE TITLE
stylecheck fix

### ISSUE THREAD

**Issue by timotheecour** [nim-compiler-dev/nimlint#14]
refs: https://forum.nim-lang.org/t/7219#45590

>> It does not convert snakes to camels.

> That's not its job, no other comparable tool (gofmt, clang-tidy, etc) does that either.

=> pefect job for `nimlint`

semantics:
* a lint task: `--styleFix:definition` to convert style of a symbol use to the exact identifier used in the symbol definition (eg:
```nim
proc c_printf() = discard
cprintf() # will be changed to c_printf
```

* a lint task: `--styleFix:caml` that would convert style of all symbols to caml

**Comment by juancarlospaco** [comment]
Ideas for things that should produce warning or error or autofix:

- `distinct tuple` type should be a code smell.
- `Positive` and `Natural` as return type should be a code smell.
- `try` blocks with >5 `except` branches produce 1 warning, with >10 `except` branches produce 1 error.
- If range is near or equal a sized integer, hint using a sized integer, `var x: range[-128..128]` suggest using `int8` instead.
